### PR TITLE
#166 - assertions for metadata in RpmITCase

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #166 I've added assertions to check metadata in `RpmITCase`.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/309#issuecomment-652505138))

@olenagerasimova Job `gh:artipie/rpm-adapter#309` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/artipie/rpm-adapter/issues/309#issuecomment-652505138))

@olenagerasimova Pull request [#309](https://github.com/artipie/rpm-adapter/issues/309) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: src/test/java/com/artipie/rpm/RpmITCase.java [hunk 1]
```diff
@@ -111,18 +110,44 @@ void tearDown() throws Exception {
...
+        ) {
+            final Path xml = this.metadataFromStorage(bsto, pckg);
+            MatcherAssert.assertThat(
+                String.format(
+                    "Metadata %s has %d rpm packages", pckg.name(), RpmITCase.SIZE.count()
+                ),
+                new XMLDocument(xml),
+                new NodeHasPkgCount(RpmITCase.SIZE.count(), pckg.tag())
+            );
+        }
```
Comment (olegmoz) [inline src/test/java/com/artipie/rpm/RpmITCase.java:129]: @olenagerasimova that is quite large chunk of code duplicated in two tests, do I'd suggest to extract a method or a matcher class from it

**Review by olegmoz — APPROVED** [review]
@olenagerasimova everything is good, got only one suggestion

File: src/test/java/com/artipie/rpm/RpmITCase.java (hunk 1 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/rpm/RpmITCase.java:129]: @olegmoz eventually I'm planning to get rid of `batchUpdateIncrementally` method, so these duplications also should be removed

**Comment by 0crat** [comment]
Job `gh:artipie/rpm-adapter#309` is not assigned, can't get performer

**Pull request merged**